### DISCUSSION TITLE
Bolt 5 : doctrine table prefix can be changed ?

### DISCUSSION THREAD

**Discussion by celtic34fr** [bolt/core#3346] [Q&A] [answered]
Hi there,

does anyone know how to change the prefix value of doctrine tables? When I created custom entities with the name directive, the name I gave was prefixed with 'bolt', I want to have another prefix to distinguish bolt core entity from entities in my application.

**Comment by abdellahrk** [comment, marked as answer]
@celtic34fr you can look into `config/services.yaml` there is a key under `parameters` call `bolt.table_prefix` if that can help.

**Reply by celtic34fr** [reply]
Yes [@abdellahrk,](https://github.com/abdellahrk) that's right, but you change the prefix for all the tables of the project : bolt core table and the project's tables.
Do you know how to change only for the project's tableS, but not after make installation ?
